### PR TITLE
feat: get bot appearance by apiToken

### PR DESCRIPTION
**Pull request by datnguyennnx**
- Get data of bot appearance by apiToken. 

- You can testing this api following code below: 

```
const { data: getBotAppearanceSettingByApiToken } =
    api.bot.getBotAppearanceSettingByApiToken.useQuery({
      apiToken: '',
    })
console.log(getBotAppearanceSettingByApiToken)
```

### CODE REVIEW

File: src/server/api/routers/bot.ts [hunk 1]
```diff
@@ -187,6 +192,30 @@ export const botRouter = createTRPCRouter({
       return bot
     }),
 
+  getBotAppearanceSettingByApiToken: integrationProcedure.query(
+    async ({ ctx }) => {
+      const sources = await db.query.bots.findFirst({
```
Comment (taynguyen) [inline src/server/api/routers/bot.ts:197]: Should not be plural.

Should select necessary fields. Shouldn't fetch all bot field.

**Review by taynguyen — COMMENTED** [review]
Minor comments.

File: src/server/api/routers/bot.ts [hunk 2]
```diff
@@ -187,6 +192,30 @@ export const botRouter = createTRPCRouter({
       return bot
     }),
 
+  getBotAppearanceSettingByApiToken: integrationProcedure.query(
+    async ({ ctx }) => {
+      const source = await db.query.bots.findFirst({
+        where: eq(schema.bots.id, ctx.session.botId),
```
Comment (taynguyen) [inline src/server/api/routers/bot.ts:198]: Let's add option columns to specify necessary fields to return.

File: src/server/api/routers/bot.ts [hunk 3]
```diff
@@ -187,6 +192,39 @@ export const botRouter = createTRPCRouter({
...
+          accentColour: true,
+        },
+        where: eq(schema.bots.id, ctx.session.botId),
+      })
+
+      const data: BotAppearance = {
+        botId: ctx.session.botId,
+        companyLogoAttachmentId: source?.companyLogoAttachmentId ?? '',
+        botAvatarAttachmentId: source?.botAvatarAttachmentId ?? '',
+        widgetName: source?.widgetName ?? 'Dwarves Bot',
```
Comment (taynguyen) [inline src/server/api/routers/bot.ts:214]: Let's not define the default value here. Default value should only be defined when created. 
Or when show to user. Let's not make default value when get from db.

**Review by taynguyen — COMMENTED** [review]
Minor comment e. 
Think we could merge after fix.

File: src/server/api/routers/bot.ts (hunk 3 above)
Comment (datnguyennnx) [reply src/server/api/routers/bot.ts:214]: You are right. It's not necessary to return the user.

**Review by taynguyen — APPROVED** [review]

**Pull request merged**